scripts/infra/cloud-instance.sh

```diff
@@ -367,6 +367,14 @@ install_rh_nvidia_drivers() {
     fi
     "${BASH_SOURCE[0]}" "$CLOUD_TYPE" scp -n "$INSTANCE_NAME" "${SCRIPT_DIR}"/nvidia-setup.sh
     "${BASH_SOURCE[0]}" "$CLOUD_TYPE" ssh -n "$INSTANCE_NAME" "sudo ./nvidia-setup.sh"
+
+    cat <<-EOF | sudo dd of=/etc/motd.d/cuda-warning
+========================================================
+Important: Avoid running \`dnf update\`, As updating the
+           kernel will invalidate the GPU drivers
```

Comment:
It is possible to fix the drivers after dnf updating.  It might just work by rerunning nvidia-setup.sh.  Or cloud-instance.sh install-rh-nvidia-drivers.  Would it be appropriate to change the message to something like:

Running \`dnf update\` may invalidate the GPU drivers.  See \`cloud-instance.sh\` for more details.

Reply:
I'll try it out, I might have to add some more context as when I hit this problem I was using the AWS image outside of instructlab   (i.e. I used cloud-instance.sh to launch the instance but not for any of the setup)

Reply:
I've updated it to add more details and actually install the motd on the correct machine :wink: 

Also rerunning install_rh_nvidia_drivers on a older image isn't working (due to package dep errors)
https://github.com/instructlab/instructlab/issues/3241
https://github.com/instructlab/instructlab/pull/3242